ModelLibrary/Trade.cs

```diff
@@ -1,14 +1,19 @@
 namespace ModelLibrary
 {
-    public class Trade(InventoryItems requestedItem, int requestedAmount, InventoryItems offeredItem, int offeredAmount, string targetName, string originName)
+    public class Trade(InventoryItems requestedItem, int requestedAmount, InventoryItems offeredItem, int offeredAmount)
     {
-        public string targetName { get; set; } = targetName;
-        public string originName { get; set; } = originName;
```

Comment:
I know i told you we don't need this, which is true for the current code. but i forgot about starting a trade with TTS which might still require this.

Reply:
Added them back.